CODEOWNERS

```diff
@@ -184,6 +184,9 @@ k8s/**/common/bar/identity.yaml @hmcts/Fees
 k8s/**/common/bar/namespace.yaml @hmcts/Fees
 
 # IDAM
+k8s/prod/common/idam/idam-api.yaml @hmcts/idam
+k8s/prod/common/idam/idam-web-public.yaml @hmcts/idam
+k8s/prod/common/idam/idam-web-admin.yaml @hmcts/idam
```

Comment:
Based on the readme, I think this is what you're after @timja

Reply:
update the aat ones to be **